source/getting-started/onboarding.markdown

```diff
@@ -5,10 +5,14 @@ description: "Instructions to get Home Assistant configured."
 
 Alright, you made it here. The tough part is done.
 
-With Home Assistant installed, it's time to configure it. Here you will create the owner account of Home Assistant. This account will be an administrator and will always be able to change everything. Enter a name, username, password and click on "create account".
+With Home Assistant installed, it's time to configure it. 
+
+If this is your first time installing Home Assistant, or if you want a fresh install keep reading. However, if you have a snapshot saved of a previous setup that you want to load, this is your chance to do so using the link at the bottom - scroll down for the details.
```

Comment:
I feel like we are out of context here.

The getting started guide, is aimed at getting started for the first time.
Considering that, putting in a restore guide in here, feels a bit out of place?

Reply:
I see your point, but in reality I found myself going back to this flow (which is one click a away from homepage rather then going to the documentation and finding the relevant page).

I don’t think it hurts to have a paragraph for people looking to recover a previous version - think of it as “getting started (all over again)” ;)

Reply:
I guess no answer means you don’t see it that way?
* Is there anything you want to keep from the above changes?
* Would you like to have a dedicated page for restoring upon starting over ?
* if so, can we have a link from this page to that new on or would that also feel out of place in your opinion?

@frenck

Reply:
No answer means, we are very busy with over 600 Prs open and receiving around 100-150 new PRs each day. It means, have some patience.